integration-tests/rest-tests-jaxrs/src/test/groovy/org/hawkular/metrics/alerting/ExternalAlerterITest.groovy

```diff
@@ -61,10 +68,12 @@ class ExternalAlerterITest extends AlertingITestBase {
         resp = client.post(path: "triggers", body: triggerTestAvg)
         assertEquals(200, resp.status)
 
-        // ADD external metrics avg condition. Note: alerterId must be "HawkularMetrics"
+        // Note: alerterId must be "HawkularMetrics"
         // Average over the last 1 minute > 50, check every minute
+        Query qNow = Query.builder("qNow").metric(metricId).duration("1mn").build();
```

Comment:
I know that groovy is unbeatable for these tests where Java objects are "JSON-lized" under the hood in a pretty straightforward way.
But experience showed us when someone else needs to use this feature, and we point them to these examples, they are not very useful, as they need to access via Java API.
I think these tests are great and we should continue using but what I do is to introduce some additional test where instead of an object we define the json representation in raw format.
That helps to catch up with the technology.